docs/source/app-dev/app-arch.rst

```diff
@@ -10,15 +10,12 @@ This section describes our recommended design of a full-stack Daml application.
 
 .. image:: ./recommended_architecture.svg
 
-The above image shows the recommended architecture. Of course there are many ways how you can change
-the architecture and technology stack to fit your needs, which we'll mention in the corresponding
-sections. Note that the Participant Node is integrated into the Daml drivers in some cases rather
-than being part of the Application Backend. See :doc:`/support/overview` for more details.
+The above image shows the recommended architecture. Of course there are many ways that the architecture and technology 
+stack can be changed to fit your needs which we'll mention in the corresponding sections.
 
 To get started quickly with the recommended application architecture, generate a new project using the ``create-daml-app`` template:
 
 .. code-block:: bash
-
```

Comment:
Sorry I wasn’t trying to delete the line but to delete the space you added. You need the empty line.